Makefile

```diff
@@ -497,23 +497,27 @@ ci-env-stop:
 build-on-debian-dev:
 	@$(call func_echo_status, "$@ -> [ Start ]")
 	$(ENV_DOCKER) build -t $(ENV_APISIX_IMAGE_TAG_NAME)-debian-dev \
+		--build-arg TARGETARCH=$(ENV_OS_ARCH) \
 		--build-arg CODE_PATH=. \
 		--build-arg ENTRYPOINT_PATH=./docker/debian-dev/docker-entrypoint.sh \
 		--build-arg INSTALL_BROTLI=./docker/debian-dev/install-brotli.sh \
 		--build-arg CHECK_STANDALONE_CONFIG=./docker/utils/check_standalone_config.sh \
 		-f ./docker/debian-dev/Dockerfile .
 	@$(call func_echo_success_status, "$@ -> [ Done ]")
 
-### push-multiarch-dev-on-debian : Push apache/apisix:dev image
-.PHONY: push-multiarch-dev-on-debian
-push-multiarch-dev-on-debian:
+.PHONY: push-on-debian-dev
+push-on-debian-dev:
 	@$(call func_echo_status, "$@ -> [ Start ]")
-	$(ENV_DOCKER) buildx build --network=host --push \
-		-t $(IMAGE_NAME):dev \
-		--platform linux/amd64,linux/arm64 \
-		--build-arg CODE_PATH=. \
-		--build-arg ENTRYPOINT_PATH=./docker/debian-dev/docker-entrypoint.sh \
-		--build-arg INSTALL_BROTLI=./docker/debian-dev/install-brotli.sh \
-		--build-arg CHECK_STANDALONE_CONFIG=./docker/utils/check_standalone_config.sh \
-		-f ./docker/debian-dev/Dockerfile .
+	$(ENV_DOCKER) tag $(ENV_APISIX_IMAGE_TAG_NAME)-debian-dev $(IMAGE_NAME):dev-$(ENV_OS_ARCH)
+	$(ENV_DOCKER) push $(IMAGE_NAME):dev-$(ENV_OS_ARCH)
+	@$(call func_echo_success_status, "$@ -> [ Done ]")
+
+### merge-dev-tags : Merge architecture-specific dev tags into a single dev tag
+.PHONY: merge-dev-tags
+merge-dev-tags:
+	@$(call func_echo_status, "$@ -> [ Start ]")
+	$(ENV_DOCKER) manifest create $(IMAGE_NAME):dev \
+		$(IMAGE_NAME):dev-amd64 \
+		$(IMAGE_NAME):dev-arm64
```

Comment:
Do we need to remove these individual images from Docker Hub?

Reply:
Doesn't it seem like a necessary operation? Just a matter of choice

Reply:
I tried searching, but it seems there isn't a related command to delete this remote tag. Only this https://docs.docker.com/reference/cli/docker/manifest/rm/

Reply:
I haven't found a faster way to provide dev image.

In order to only upload the dev tag image, I seem to have to cache and download the built image to the current job, then upload it.